xcell/tests/test_mapper_SDSS.py

```diff
@@ -0,0 +1,98 @@
+import numpy as np
```

Comment:
This is great. I think these probably supersede the `eBOSS` tests, so you should `git rm` that file.

Reply:
Done